### PR TITLE
[WIP] New & Updated PR for SRVLOGIC-165-New-midstream: Add a TP and DP notes and change the title to Technology and Developer Preview releases

### PR DESCRIPTION
**Pull request by kaldesai**
**Tracking JIRA:** https://issues.redhat.com/browse/SRVLOGIC-165

**Doc previews & Description:** 
Please check the following sections: 

The overview conveys that this URL covers information that is DP and TP and the respective section mentions that they are TP or DP respectively. 

- Changed the title of the doc: **OpenShift Serverless - Technology and Developer Preview Releases**

- [Overview](https://deploy-preview-104--jazzy-shortbread-5f62b7.netlify.app/docs/latest/) (**The overview covers the note about both TP and DP**)

- [About OpenShift Serverless Logic](https://deploy-preview-104--jazzy-shortbread-5f62b7.netlify.app/docs/latest/serverless-logic/about) (**Added a TP note**)

- [Setup OpenShift Serverless with OpenShift Service Mesh](https://deploy-preview-104--jazzy-shortbread-5f62b7.netlify.app/docs/latest/serverless/service-mesh/common-service-mesh-setup) (**Added a TP note**)

- [About buildpacks for OpenShift Serverless Functions](https://deploy-preview-104--jazzy-shortbread-5f62b7.netlify.app/docs/latest/functions/serverless-functions-about) (**Added a DP note**)

- [Use Serving with OpenShift ingress sharding](https://deploy-preview-104--jazzy-shortbread-5f62b7.netlify.app/docs/latest/serverless/serving/serving-with-ingress-sharding) (**Added a DP note**)

### CODE REVIEW

**Comment by kaldesai** [comment]
@mgencur, I have incorporated all of your suggestions here in this new PR. Can you please take a look and verify once? (Closed the previous PR)

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -2,16 +2,18 @@
...
+{serverlessproductname} - Technology and Developer Preview releases contain features and functionalities that might not be fully tested. Customers are encouraged to provide feedback on Technology and Developer Preview releases.
 
-Developer Preview releases are not production-ready, and customers are recommended to avoid using the project for production or business-critical workloads.
+Technology and Developer Preview releases are not production-ready, and customers are recommended to avoid using the project for production or business-critical workloads.
 
-If you need assistance with OpenShift Serverless - Developer Preview releases, please reach out on the following mailing list, where the Red Hat Development Team will be happy to assist you based on the availability and work schedules: serverless-interest@redhat.com
-====
+{serverlessproductname} provides information about additional, advanced use cases and integrations that are possible using OpenShift Container Platform and OpenShift Serverless, but are for technology and development purposes and are not officially supported by Red Hat.
```
Comment (mgencur) [inline modules/ROOT/pages/index.adoc:9]: I think OpenShift Container Platform should be replaced with some variable `{ ... }` and OpenShift Serverless should be replaced with `{serverlessproductname}`.

The rest of this PR looks good. Thanks

Comment (kaldesai) [reply modules/ROOT/pages/index.adoc]: Thank you so much! I have added the attribute for the OpenShift Container Platform.

Comment (kaldesai) [reply modules/ROOT/pages/index.adoc]: @mgencur Can you please merge this PR now if everything is fine?

**Pull request merged**